### ISSUE TITLE
Authentication and Authorization for CCX REST API

### ISSUE THREAD

**Issue by pdpinch** [mitocw/edx-platform#158]
Needs to be scoped to an API user that has the necessary roles for creating a CCX.

**Comment by pdpinch** [comment]
We can't open #145 upstream on edX until this is resolved.

**Comment by pdpinch** [comment]
We can re-use an existing role, such as instructor on the master course.

**Comment by pdpinch** [comment]
merged into Giovanni's branch
